### PR TITLE
Public docs update (e110) - VEP

### PR DESCRIPTION
**Pull request by nakib103**
Sandbox test urls - 
http://wp-np2-11.ebi.ac.uk:7070/info/docs/tools/vep/script/vep_cache.html
http://wp-np2-11.ebi.ac.uk:7070/info/docs/tools/vep/script/vep_plugins.html
http://wp-np2-11.ebi.ac.uk:7070/info/docs/tools/vep/script/vep_download.html#new
http://wp-np2-11.ebi.ac.uk:7070/info/docs/tools/vep/online/VEP_web_documentation.pdf
http://wp-np2-11.ebi.ac.uk:7070/info/docs/tools/vep/script/VEP_script_documentation.pdf

### CODE REVIEW

**Review by nuno-agostinho — CHANGES_REQUESTED** [review]
Hey @nakib103, thanks for the update!

I noticed some plugin documentations was not the best, so I created a PR to improve on this: https://github.com/Ensembl/VEP_plugins/pull/598.

Also, https://github.com/Ensembl/VEP_plugins/pull/595 will change the docs for PON plugin, so we have to wait until that is merged before making this PR ready to merge.

**Comment by nuno-agostinho** [comment]
Hey @nakib103, the previous PRs were already merged. However, PR Ensembl/VEP_plugins#601 also impacts VEP plugin docs. Maybe we can wait until this is merged to update plugin docs, but tell me if you think otherwise. Thanks!

**Comment by nakib103** [comment]
Yes, I will wait for that PR to be merged and re-generate the plugin docs.

File: docs/htdocs/info/docs/tools/vep/script/vep_download.html
Comment (nuno-agostinho) [inline docs/htdocs/info/docs/tools/vep/script/vep_download.html:126]:
```suggestion
        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/MaveDB.pm" rel="external">MaveDB</a> - adds information from MaveDB, a database that holds experimentally determined measures of variant effect</li>
```

**Review by nuno-agostinho — APPROVED** [review]
Looks good to me, thanks @nakib103!

**Pull request merged**
